### PR TITLE
Add support for font-size to LineEdit and Text-Edit widgets.

### LINKED ISSUE #435: Add support for `font-size` to `LineEdit` in all styles
### Discussed in https://github.com/sixtyfpsui/sixtyfps/discussions/434

<div type='discussions-op-text'>

<sup>Originally posted by **LouisGariepy** August 26, 2021</sup>
Hello! First of all, thanks for this wonderful project, I am falling in love with it :smile: .

I was wondering if there is a way to change the font size for this widget. Since there are width/height properties, it is natural to assume that there is a property to change the font size, but that doesn't seem to be the case.

Thanks!</div>

### PR DESCRIPTION
**Pull request by besok**
closes #435

### CODE REVIEW

**Comment by ogoffart** [comment]
Thanks for the patch, the implementation looks good. but please clean it up a bit, don't modify the ignore file or the 7gui example. 
Appart from that, looks good to me

File: docs/widgets.md [hunk 1]
```diff
@@ -181,6 +182,7 @@ import { LineEdit } from "sixtyfps_widgets.60";
 Example := Window {
     width: 200px;
     height: 25px;
+    font-size: 14px;
```
Comment (ogoffart) [inline docs/widgets.md:185]: That's in Window, not in the LineEdit

**Review by ogoffart — CHANGES_REQUESTED** [review]

File: docs/widgets.md (hunk 1 above)
Comment (besok) [reply docs/widgets.md:185]: right

**Comment by ogoffart** [comment]
The commit of the pull request are not associated with your github account. Probably because the email doesn't match.
You need to make sure that the email that you used to make the commit is associated with your github account.  Or you can rebase and change the email in the commit to match the one associated with the github account.

**Review by ogoffart — APPROVED** [review]

**Comment by ogoffart** [comment]
Thank you!

**Pull request merged**